### ISSUE TITLE
horrible slow sync on pi3b+

### ISSUE THREAD

**Issue by phenomeus** [dgarage/NBXplorer#209]
hi, 

I'm suffering right now to get nbxplorer to sync up to the btc node (same pi) to start using btcpayserver. I know the hardware is very low but maybe there is a trick to speed up the sync. right now I'm restarting the nbxplorer.service every 10 minutes just to free up the ram and proceed with syncing because otherwise its hanging the whole pi because of 700mb+ ram usage out of 1024 and the 2GB swap file is also getting filled 

hope to get through soon with some ideas or just my workaround

**Comment by pavlenex** [comment]
This is already documented in official[ btcpay RPI3 docs,](https://github.com/btcpayserver/btcpayserver-docker/tree/master/contrib/FastSync) use[ fast sync.](https://github.com/btcpayserver/btcpayserver-docker/tree/master/contrib/FastSync)

**Comment by phenomeus** [comment]
yeah I know but its not recommended. and I'm catching up: 601959 / 605903

so no other suggestions only fast sync right?

**Comment by pavlenex** [comment]
What do you mean it's not recommended? It's the only way to make it work on RPI3 afaik.

**Comment by phenomeus** [comment]
https://github.com/btcpayserver/btcpayserver-docker/tree/master/contrib/FastSync#what-are-the-downsides-of-fast-sync

**Comment by NicolasDorier** [comment]
just follow https://github.com/btcpayserver/btcpayserver-docker/tree/master/contrib/FastSync#how-to-verify-that-the-utxo-set-snapshot-follows-the-consensus then

**Comment by phenomeus** [comment]
I bought a pi 4 4GB yesterday, I will retry it with the beefier hardware, but thanks
